### ISSUE TITLE
Cannot import extension autohttp.flask with sphinx v1.7.1

### ISSUE THREAD

**Issue by mzanini** [sphinx-contrib/httpdomain#5]
Recent versions of Sphinx have removed the Directive class from sphinx.util.compat, trying to import the autohttp.flask extension produces this error:
```
sphinx-build -b html -E -T source build
Running Sphinx v1.7.1

Traceback (most recent call last):
  File "/usr/lib64/python2.7/site-packages/sphinx/cmdline.py", line 303, in main
    args.warningiserror, args.tags, args.verbosity, args.jobs)
  File "/usr/lib64/python2.7/site-packages/sphinx/application.py", line 191, in __init__
    self.setup_extension(extension)
  File "/usr/lib64/python2.7/site-packages/sphinx/application.py", line 411, in setup_extension
    self.registry.load_extension(self, extname)
  File "/usr/lib64/python2.7/site-packages/sphinx/registry.py", line 318, in load_extension
    raise ExtensionError(__('Could not import extension %s') % extname, err)
ExtensionError: Could not import extension sphinxcontrib.autohttp.flask (exception: cannot import name Directive)
```

This works with Sphinx v.1.6.1.

**Comment by dahlia** [comment]
This actually was fixed by PR #4, but not released yet.

**Comment by dahlia** [comment]
Released a new version: <https://pypi.org/project/sphinxcontrib-httpdomain/1.6.1/>.
